file-formats/susi/type/zif_aff_susi_v1.intf.abap

```diff
@@ -0,0 +1,175 @@
+INTERFACE zif_aff_susi_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Maintenance Mode</p>
+    "! Maintenance Mode
+    "! $values {@link if_aff_susi_v1.data:co_maintenance_mode}
```

Comment:
I think the "z" was lost during last update. This might also fix the failed job
```suggestion
    "! $values {@link zif_aff_susi_v1.data:co_maintenance_mode}
```

Reply:
done